### PR TITLE
Change ccompat api global id 796

### PR DESCRIPTION
**Pull request by carlesarnal**

### CODE REVIEW

File: app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java [hunk 1]
```diff
@@ -71,13 +71,11 @@
     }
 
     @Override
-    public List<SubjectVersion> getSubjectVersions(int globalId) {
+    public List<SubjectVersion> getSubjectVersions(int contentId) {
```
Comment (famartinrh) [inline app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java:74]: @carlesarnal @EricWittmann this facade is shared with the IBM api no? will it be an issue for ibm if we change this to use contentId

Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java:74]: Is it really?  I didn't realize that.  If that's true then yes that would be an issue for them.

Comment (carlesarnal) [reply app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java:74]: I can't see this facade being used in the IBM API. We're directly injecting the storage. @famartinrh do you have a reference to that?

Comment (EricWittmann) [reply app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java:74]: I've never been happier that @famartinrh was wrong.  :)  Whew!!

**Comment by EricWittmann** [comment]
This looks all good to me - just waiting on a resolution to the build failure.  Please someone let me know when this is passing tests and ready to be merged. :)

**Comment by EricWittmann** [comment]
This looks all good to me - just waiting on a resolution to the build failure.  Please someone let me know when this is passing tests and ready to be merged. :)

**Review by famartinrh — APPROVED** [review]
Finally!!

**Pull request merged**